lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -111,13 +112,19 @@ public APIGatewayProxyResponseEvent handleRequest(
                     gpg45ProfileEvaluator.contraIndicatorsPresent(
                             evidenceMap, clientSessionDetailsDto);
             if (contraIndicatorErrorJourneyResponse.isEmpty()) {
-                updateSuccessfulVcStatuses(ipvSessionItem, credentials);
```

Comment:
We've lost the call to `updateSuccessfulVcStatuses`. Does that matter?

Reply:
Its just moved down below now

Reply:
ha, oh yeah! sorry